src/utils/remote-editor.js

```diff
@@ -9,11 +9,16 @@ import apiFetch from '@wordpress/api-fetch';
 import { getGBKit } from './bridge';
 import { error } from './logger';
 
+/**
+ * Cache for editor assets to avoid unnecessary network requests
+ * @type {Object|null}
+ */
+let editorAssetsCache = null;
+
 /**
  * @typedef {Object} EditorAssetConfig
  *
  * @property {string[]} allowedBlockTypes Array of allowed block types provided by the API.
- * @property {Object}   wpDependencies    WordPress dependencies, empty when allowedPackages is provided.
```

Comment:
This was removed in a separate commit as its usage was removed entirely in an earlier commit.